Refactor code to Eslint 9.x (#3375)
mitre/saf: README.md

## Terminology

> - ["Heimdall"](https://github.com/mitre/heimdall2) - Our visualizer for all security result data

Reviewer: ```suggestion
- ["Heimdall"](https://github.com/mitre/heimdall2) - A visualizer for all security result data
```

> - ["OASIS Heimdall Data Format (OHDF) - aka HDF"](https://saf.mitre.org/#/normalize) - Our common data format to preserve and transform security data

Reviewer: ```suggestion
- ["OASIS Heimdall Data Format (OHDF) - aka HDF"](https://saf.mitre.org/#/normalize) - A common data format to preserve and transform security data
```

Author: like it - resolved

>[!NOTE]

> > All mentioning of HDF in this document references the OHDF

Reviewer: ```suggestion
> All mention of HDF in this document refers to the OHDF.
```

Author: got it - done

      * [Update via Windows Installer](#update-via-windows-installer)

## Developers 

> For detail information about development, testing , and contributing to the SAF project refer to [MITRE SAF Development](https://github.com/mitre/saf/blob/main/docs/contributors-guide.md)

Reviewer: ```suggestion
For detailed information about development, testing , and contributing to the SAF project refer to [MITRE SAF Development](https://github.com/mitre/saf/blob/main/docs/contributors-guide.md)
```

Author: done

  * [Attest](#attest)
      * [Create Attestations](#create-attestations)
      * [Apply Attestations](#apply-attestations)

> ### [Get Help with Converts](#converts)

Reviewer: ```suggestion
### [Get Help with Convert](#convert)
```

To me it makes more sense without the s. We could also be even more clear and say:
### [Get Help with Convert Commands](#convert)

  $ saf attest apply -i hdf1.json hdf2.json attestation.xlsx -o outputDir
```
[top](#usage)
### Converts

Translating your data to and from Heimdall Data Format (HDF) is done using the `saf convert` command.

Want to Recommend or Help Develop a Converter? See [on how to get started](https://github.com/mitre/saf/wiki/How-to-recommend-development-of-a-mapper) 📰

> [top](#get-help-with-converts)

Reviewer: ```suggestion
[top](#get-help-with-convert)
```
